### PR TITLE
F add camera calibration

### PR DESCRIPTION
**Pull request by beatrizleon**

### CODE REVIEW

File: include/image_publisher.h [hunk 1]
```diff
@@ -23,7 +23,7 @@ namespace Pylon
 
     ImagePublisher(ros::NodeHandle nh, sensor_msgs::CameraInfo::Ptr cinfo, string frame_id)
       : nh_(nh), it_(nh_) {
-      cam_pub_ = it_.advertiseCamera("camera/image_raw", 1);
+      cam_pub_ = it_.advertiseCamera(frame_id+"/image_raw", 1);
```
Comment (AndriyPt) [inline include/image_publisher.h]: it will look nicer with spaced before and after + sign :)

File: src/basler_camera_node.cpp [hunk 2]
```diff
@@ -197,6 +203,15 @@ int main(int argc, char* argv[])
 
     handle_basler_parameters(camera);
 
+    // Set the pixel format to RGB8 if available.
+    INodeMap& nodemap = camera.GetNodeMap();
+    CEnumerationPtr pixelFormat( nodemap.GetNode( "PixelFormat"));
+    String_t oldPixelFormat = pixelFormat->ToString();
+    if ( IsAvailable( pixelFormat->GetEntryByName( "RGB8")))
```
Comment (AndriyPt) [inline src/basler_camera_node.cpp]: either put space before closing parenthesis or remove space after opening parenthesis

File: src/basler_camera_node.cpp [hunk 3]
```diff
@@ -197,6 +203,15 @@ int main(int argc, char* argv[])
...
+    {
+        pixelFormat->FromString( "RGB8");
```
Comment (AndriyPt) [inline src/basler_camera_node.cpp]: remove space after opening parenthesis

**Comment by beatrizleon** [comment]
@AndriyPt I fixed your comments

**Pull request merged**